### ISSUE TITLE
Downloaded dataset can not be applied

### ISSUE THREAD

**Issue by GrayChan04** [GEO-optim/GEO#1]
Hello! After I ran the code, it prompted that there was no test set. Then I saw on huggingface that there was indeed only a training set. The 'sugg_idx' field did not exist in the training set, so I could not run the code. How can I adjust the dataset to run the code?

**Comment by jackwestbrock** [comment]
I am having the same issue @GrayChan04 , were you able to figure it out?

**Comment by HaritzPuerto** [comment]
I am having the same issue. Any plans to fix it? Thanks!

**Comment by Pranjal2041** [comment]
Hi, the issues should be resolved now. [7102d71](https://github.com/GEO-optim/GEO/commit/7102d71ba92ec5e443e8ce76237840e381dda910)
